pangeo-cnes.tex

```diff
@@ -83,75 +99,94 @@ \subsection{Motivations, mission and goals}
-
-Modules to automatically deploy distributed clusters in various infrastructures are being developed: dask-kubernetes for Kubernetes clusters and thus the public cloud, dask-jobqueue\cite{b4} for HPC systems using scheduler such as Slurm, PBS Pro or LSF, and dask-yarn for YARN clusters (i.e. Hadoop).
-
 \begin{figure}
   \centering
   \includegraphics[width=\columnwidth]{pangeo_stack.png}
   \caption{\label{pangeo_stack} Pangeo platform main components.}
 \end{figure}
 
+There are several developments that either started in or are fueled by the
+Pangeo community:
+\begin{itemize}
+\item Modules to automatically deploy Dask distributed clusters in various infrastructures are being developed: dask-kubernetes for Kubernetes clusters and thus for the public cloud, dask-jobqueue\cite{b4} for HPC systems using scheduler such as Slurm, PBS Pro or LSF, and dask-yarn for YARN clusters (i.e. Hadoop).
+\item The integration of a Zarr backend in xarray paved the way to directly access cloud-based object storage in parallel computations.
+\item ...
+\end{itemize}
+
+
+\subsection{Deployments}
+\label{ssec:deployments}
+
+The GitHub community offers online documentation, scripts and other tools to link them together in order to deploy a Pangeo platform (see \ref{pangeo_stack}) and put the software stack on HPC systems or in the public cloud. The main elements allowing to build and use the platforms along the core libraries are first a set of scripts and documentation that allows automatically creating the necessary cloud infrastructure. Currently available for Google Cloud Engine (GCE), and work is underway for Amazon Web Services (AWS) compatibility. Terraform software and CI/CD tooling are also under active development and offer the potential for automating these tasks.
+
+
 \subsection{Applications}
 \label{ssec:applications}
 
-Pangeo's first scientific domain is earth sciences: particularly the domains of oceanographic, terrestrial, atmospheric, and climate sciences. There are already many applications documented in these fields. Some real science use cases can be found online\cite{b5}; these examples, along with their data, can be directly executed from a web browser. We can for example cite \textit{Sea Surface Altimetry Data Analysis} that explore the increase of global mean sea level over 20 years of data, or also \textit{US Precipitation and Temperature Analysis} that explore meteorological gridded ensemble precipitation and temperature estimates over the contiguous United States.
+Pangeo's first scientific domain are the earth-system sciences. There are already many applications documented in these fields. Some real science use cases can be found online\cite{b5}; these examples, along with their data, can be directly executed on cloud infrastructure and be controlled from a web browser.
```

Comment:
these examples ~~, along with their data,~~ can be directly executed on cloud infrastructure ~~and be controlled~~ from a web browser.

data is not executed

Reply:
Sounds fair